substrate/frame/staking-async/src/pallet/mod.rs

```diff
@@ -1388,24 +1362,12 @@ pub mod pallet {
 		/// Emits `Withdrawn`.
 		///
 		/// See also [`Call::unbond`].
-		///
-		/// ## Parameters
-		///
-		/// - `num_slashing_spans` indicates the number of metadata slashing spans to clear when
-		/// this call results in a complete removal of all the data related to the stash account.
-		/// In this case, the `num_slashing_spans` must be larger or equal to the number of
-		/// slashing spans associated with the stash account in the [`SlashingSpans`] storage type,
-		/// otherwise the call will fail. The call weight is directly proportional to
-		/// `num_slashing_spans`.
 		#[pallet::call_index(3)]
-		#[pallet::weight(T::WeightInfo::withdraw_unbonded_kill(*num_slashing_spans))]
-		pub fn withdraw_unbonded(
-			origin: OriginFor<T>,
-			num_slashing_spans: u32,
```

Comment:
Can you please leave the argument here to avoid the transaction needing to be re-implemented in wallets, and just update the docs that it is not used + will be deprecated later? 

I know we are taking this new fork as an oppurtunity to clean up as much of the old code as possible, but keeping it as-backwards-compatible-as-possible is still a goal that I find important. 

In other words, I would highly prefer if we can tell to all wallets upon AHM: all the user-facing transactions is the same with `pallet-staking`, just the pallet name has changed.

Reply:
Ok yeah fair point. Will do.

Reply:
The benchmarks, and all the internal functions can already start to ignore this, or not have it at all.